### PR TITLE
feat: Add Arabic translations

### PR DESCRIPTION
**Pull request by EliasAfara**
## Summary

Added Arabic translations

Reference #48 

## Type of change

- [x] New feature (added a non-breaking change which adds functionality)

### CODE REVIEW

File: api/locale/ar.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ar:
+  direction: rtl
+  view: "1 مشاهدة"
+  views: "%{number} مشاهدات"
```
Comment (DenverCoder1) [inline api/locale/ar.yml:4]: Thanks!

It seems like the YouTube interface uses `مشاهدة` for multiple views, do you know what the difference might be?

Comment (EliasAfara) [reply api/locale/ar.yml:4]: YouTube in English and Arabic formats view counts differently. In English, YouTube uses abbreviations like "M" for millions and "k" for thousands to make large numbers more concise and readable, as shown below:

- 5.6M views (5.6 million views)
- 12k views (12 thousand views)

This format is common in English and helps users quickly grasp the magnitude of view counts without reading all the digits.

On the other hand, in Arabic, YouTube uses the full form of numbers without such abbreviations, making it slightly longer but still clear and easily understandable for Arabic-speaking users, as seen in these examples:

- 236 ألف مشاهدة (236 thousand views)
- 1.1 مليون مشاهدة (1.1 million views)

The difference in formatting view counts between English and Arabic on YouTube is not limited to numeric representation. In Arabic, "مشاهدة" (singular form) is used consistently in all cases, regardless of the number of views. This choice is made for consistency and simplicity in the user interface, as Arabic has different forms for singular, dual, and plural quantities, and matching them with numbers in different ranges can be complex. Using the singular form ensures a clear and straightforward presentation of view counts for Arabic-speaking users, maintaining a consistent user experience.

Comment (DenverCoder1) [reply api/locale/ar.yml:4]: So which do you think is better?

This is what we currently have it looking like with your translation:

https://ytcards-git-fork-eliasafara-yt-cards-arabic-10c6e6-denvercoder1.vercel.app/?id=6u9BrDaSHJc&title=Automatically+Deploy+to+Fly.io+with+GitHub+Actions&lang=ar&timestamp=1661889604&background_color=%230d1117&title_color=%23ffffff&stats_color=%23dedede&width=250&border_radius=5&duration=312&max_title_lines=1

Do you think Google has it right and it should be changed to be consistent, or do you think the way it is is better?

Comment (EliasAfara) [reply api/locale/ar.yml:4]: let's change it

```yml
ar:
  direction: rtl
  view: "1 مشاهدة"
  views: "%{number} مشاهدة"
```

**Review by DenverCoder1 — APPROVED** [review]
Looks great! Thanks for the contribution :tada:

**Pull request merged**